test/constants/thatlowcarblifeConstants.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+    testUrl: "https://thatlowcarblife.com/chicken-bacon-ranch-pizza/",
+    invalidUrl: "https://thatlowcarblife.com/notarealurl",
+    invalidDomainUrl: "www.invalid.com",
+    nonRecipeUrl: "https://thatlowcarblife.com/about/",
+    expectedRecipe: {
+        name: 'Chicken Bacon Ranch Pizza',
+        ingredients: [
+            '2 cups shredded mozzarella',
+            '¾ cup almond flour',
+            '2 tablespoons cream cheese',
+            '1 egg',
+            '1 teaspoon salt',
+            '⅔ cup ranch dressing',
+            '2 cups shredded cheddar',
+            '1 cup chopped cooked chicken',
+            '6 strips bacon, fried and crumbled',
+            '2 tablespoons chopped chives'
+        ],
+        instructions: [
```

Comment:
nicer to trim them :)

Reply:
Sometimes I give the the 'ol karate chop 🥋